app/models/support/requests/content_change_request.rb

```diff
@@ -22,6 +22,13 @@ def self.label
       def self.description
         "Request changes to GOV.UK content managed by GDS content designers"
       end
+
+      def is_legal_deadline_options
```

Comment:
Nitpick: I'd rename the commit to `Add "Is this a legal deadline?" question`, as "Add radio button" is a bit vague.